pkg/provider/github/parse_payload.go

```diff
@@ -524,7 +532,42 @@ func (v *Provider) handleIssueCommentEvent(ctx context.Context, event *github.Is
 	}
 
 	v.Logger.Infof("issue_comment: pipelinerun %s on %s/%s#%d has been requested", action, runevent.Organization, runevent.Repository, runevent.PullRequestNumber)
-	return v.getPullRequest(ctx, runevent)
+	pr, err := v.getPullRequest(ctx, runevent)
+	if err != nil {
+		return nil, err
+	}
+
+	commentBody := event.GetComment().GetBody()
+	if opscomments.IsOkToTestComment(commentBody) && v.pacInfo.RequireOkToTestSHA {
+		shaFromCommentRaw := opscomments.GetSHAFromOkToTestComment(commentBody)
+		if shaFromCommentRaw == "" {
+			v.Logger.Errorf(errSHANotProvided)
+			if err := v.CreateComment(ctx, runevent, fmt.Sprintf(errSHANotProvidedComment, pr.SHA), ""); err != nil {
```

Comment:
```suggestion
			if err := v.CreateComment(ctx, runevent, fmt.Sprintf(errSHANotProvidedComment, pr.SHA), commnetTemplate); err != nil {
```

I think we should provider comment template as well to avoid multiple comments

Reply:
I think in this case it's fine as long the user would learn how to post it properly? we can do template, i just find it awkward on long pull request that is shown at the beginning of the comments and user has no idea what's going on.